main.js

```diff
@@ -0,0 +1,52 @@
+
+snakewaterMontana = {
+    guestOfHonor : "Dr. Alan Grant" ,
+    paleontologist: "Dr. Alan Grant",
+    depth: "10 meters",
+    specimen: "Velociraptor"
+  }
+    var cleverGirl = snakewaterMontana[specimen];
```

Comment:
yes you can do it like this but don't forget the `"` so what you should write is  `var cleverGirl = snakewaterMontana["specimen"];`